internal/credentials/credentials_test.go

```diff
@@ -28,78 +30,87 @@ func TestParseCredentials(t *testing.T) {
 }
 
 func TestWriteReadDeleteSystem(t *testing.T) {
-	CFG.FsRoot = t.TempDir()
-	_, err := getCredentials()
+	fsRoot := t.TempDir()
+	sysCredsPath := SystemCredentialsPath(fsRoot)
+	_, err := ReadCredentials(sysCredsPath)
 	if err != ErrMissingCredentialsFile {
 		t.Fatalf("Expected [%s], got [%s]", ErrMissingCredentialsFile, err)
 	}
-	if err := writeSystemCredentials("user1", "pass1", "1234"); err != nil {
+	if err := CreateCredentials("user1", "pass1", "1234", sysCredsPath); err != nil {
 		t.Fatalf("Unexpected error: %s", err)
 	}
-	c, err := getCredentials()
+	c, err := ReadCredentials(sysCredsPath)
 	if err != nil {
 		t.Fatalf("Unexpected error: %s", err)
 	}
 	if c.Username != "user1" || c.Password != "pass1" || c.SystemToken != "1234" {
-		t.Errorf("Unexpected user1 and pass1. Got: %s and %s",
+		t.Errorf("Expected user1 and pass1. Got: %s and %s",
 			c.Username, c.Password)
 	}
-	if err := removeSystemCredentials(); err != nil {
+	if err := util.RemoveFile(sysCredsPath); err != nil {
 		t.Errorf("Unexpected error: %s", err)
 	}
-	path := systemCredentialsFile()
-	if fileExists(path) {
-		t.Error("File was not deleted: ", path)
+	// TODO : Move to util package
```

Comment:
The comment is no longer needed.

Reply:
It looks like we're testing if the `util.RemoveFile` method is doing it’s job correctly. But it makes more sense to have a test for the RemoveFile method itself within the util package instead of repeatedly checking if it works in this file. So just leaving a note to make sure we handle it in the next stages of refactoring

Reply:
Added more context in the commit message